src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13093,6 +13096,8 @@ void VRFTerminalUnitEquipment::CalcVRF_FluidTCtrl(EnergyPlusData &state,
         }
     }
     // calculate sensible load met using delta enthalpy
+    AirMassFlow = state.dataDXCoils->DXCoil(this->HeatCoilIndex).InletAirMassFlowRate;
```

Comment:
I assume this fixes one of the issues, since AirMassFlow gets set above at line 13079.

Reply:
yeah. Somehow the air mass flow used in the coil calculation didn't get passed here correctly, that's why the heating coil heating rate and the terminal heating rate didn't match.